src/include/btree_inline.h

```diff
@@ -18,6 +18,20 @@ __wt_ref_is_root(WT_REF *ref)
     return (ref->home == NULL);
 }
 
+#ifdef HAVE_REF_TRACK
+/*
+ * __ref_save_state --
+ *     Record the state change of a ref when REF_TRACK is enabled. This function wraps the
+ *     WT_REF_SAVE_STATE macro so we can suppress it in our TSan ignore list.
+ */
+static inline void
+__ref_save_state(
```

Comment:
This is wrapping a macro to suppress TSAN but the macro is called without the wrapper. Whats up with that?